x-pack/test/security_solution_endpoint_api_int/apis/endpoint_artifacts/blocklists.ts

```diff
@@ -24,7 +25,9 @@ export default function ({ getService }: FtrProviderContext) {
   const endpointPolicyTestResources = getService('endpointPolicyTestResources');
   const endpointArtifactTestResources = getService('endpointArtifactTestResources');
 
-  describe('Endpoint artifacts (via lists plugin): Blocklists', () => {
+  describe('Endpoint artifacts (via lists plugin): Blocklists', function () {
+    targetTags(this, ['@ess', '@serverless']);
```

Comment:
Just curious - could this have been implemented using `this.targetTags()`?  That could make the implementation a bit cleaner. 

I know that `this.tag()` already exists (or used to - it was used to break up tests into groups).

Reply:
I agree on that, it would be cleaner. I found some clues about how mocha is decorated in our environment, but couldn't make this work in a short amount of time, so now I'd go with this solution, and maybe return to this later (e.g. on OnWeek)